src/objectives/elbo/entropy.jl

```diff
@@ -37,3 +37,8 @@ function estimate_entropy(
         -logpdf(q, mc_sample)
     end
 end
+
+function estimate_entropy_maybe_stl(entropy_estimator::AbstractEntropyEstimator, samples, q, q_stop)
+    q_maybe_stop = maybe_stop_entropy_score(entropy_estimator, q, q_stop)
+    estimate_entropy(entropy_estimator, samples, q_maybe_stop)
+end
```

Comment:
I agree with moving this here. But wouldn't the original function in `repgradelbo.jl` need to be removed as well?